deployLOCAL/runETL.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  9 22:45:43 2019
+
+@author: zackkingbackup
+"""
+import sys
+sys.path.append('/Users/zackkingbackup/Documents/Job Search/CapitalOne/sample-project-data-engineer/src/')
+from Extract import extract_function
+from Transform import transform_function
+from Load import load_function
+import argparse
+import boto3
+import os
+
+''' use argparse.parser to pick off command line args '''
```

Comment:
👍 for docstrings!  Great job on documenting your code.

Reply:
Thanks!